technitium-dns/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: technitium-dns
+category: networking
+name: Technitium DNS server
+version: "12.1"
+tagline: Block ads & malware at DNS level for your entire network!
+description: >-
+  Technitium DNS Server is an open source authoritative as well as recursive DNS server that can be used for self hosting a DNS server for privacy & security. It works out-of-the-box with no or minimal configuration and provides a user friendly web console accessible using any modern web browser.
+
+developer: Shreyas Zare
+website: https://technitium.com/dns/
+dependencies:
```

We just need to close this with an empty array:
```suggestion
dependencies: []
```